src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -183,6 +187,17 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, final Build
         if (intendToMerge && canMerge) {
             logger.println("Merging the pull request");
 
+			try {
+				Field ghRootField = GHIssue.class.getDeclaredField("root");
+				ghRootField.setAccessible(true);
+				Object ghRoot = ghRootField.get(pr);
+				Method anonMethod = GitHub.class.getMethod("isAnonymous");
+				anonMethod.setAccessible(true);
+				Boolean isAnonymous = (Boolean) (anonMethod.invoke(ghRoot));
+				logger.println("Merging PR[" + pr + "] is anonymous: " + isAnonymous);
```

Comment:
@DavidTanner So this ends up printing: 
`2015-09-26 08:31:23.161: Merging PR[org.kohsuke.github.GHPullRequest@30276161] is anonymous: true`

Somewhere, somehow, we're picking an anonymous GitHub for PR.